dango/account-safe/src/execute.rs

```diff
@@ -277,6 +278,10 @@ mod tests {
                     // Address below don't matter for this test.
                     ibc_transfer: Addr::mock(0),
                     oracle: Addr::mock(1),
+                    lending: Addr::mock(0), // doesn't matter for this test
+                },
+                lending: LendingAppConfig {
+                    collateral_powers: btree_map! {},
                 },
```

Comment:
For these we can implement `Default` trait on `AppConfig`, then simply do:

```rust
AppConfig {
    addresses: AppAddresses {
        account_factory: ACCOUNT_FACTORY,
        ..Default::default()
    },
    ..Default::default()
}
```

Reply:
Mm yes I thought the same. Suggested it in Discord